### ISSUE TITLE
BGP advertised-routes check

### ISSUE THREAD

**Issue by devasmith** [tynany/frr_exporter#10]
Hello. Is it possible for you to implement a function so that we can check what routes are being advertised for a prefix with frr_exporter? Example output with vtysh:
`vtysh -c 'sh ip bgp nei <prefix> advertised-routes'`

**Comment by tynany** [comment]
I am not following. Can you provide examples of the metric you're hoping to achieve?

For example:
Output:

**Comment by tynany** [comment]
I am not following. Can you provide examples of the metric you're hoping to achieve?

For example:
Output:
```
# sh ip bgp vrf blue neighbors 1.1.1.1 advertised-routes
BGP table version is 38708, local router ID is 10.110.32.3
Status codes: s suppressed, d damped, h history, * valid, > best, = multipath,
              i internal, r RIB-failure, S Stale, R Removed
Origin codes: i - IGP, e - EGP, ? - incomplete

   Network          Next Hop            Metric LocPrf Weight Path
*> 192.168.0.0/24  0.0.0.0                  0    100  64512 ?
*> 192.168.0.0/24  0.0.0.0                  0    100  64512 ?
*> 192.168.0.0/24  0.0.0.0                  0    100  64512 ?
*> 192.168.0.0/24  0.0.0.0                  0    100  64512 ?
```

**Comment by tynany** [comment]
I am not following. Can you provide examples of the metric you're hoping to achieve?

For example, with the below output, what metrics would you expect to see?

```
# sh ip bgp vrf blue neighbors 1.1.1.1 advertised-routes
BGP table version is 38708, local router ID is 1.1.1.1
Status codes: s suppressed, d damped, h history, * valid, > best, = multipath,
              i internal, r RIB-failure, S Stale, R Removed
Origin codes: i - IGP, e - EGP, ? - incomplete

   Network          Next Hop            Metric LocPrf Weight Path
*> 192.168.0.0/24  0.0.0.0                  0    100  64512 ?
*> 192.168.1.0/24  0.0.0.0                  0    100  64512 ?
*> 192.168.2.0/24  0.0.0.0                  0    100  64512 ?
*> 192.168.3.0/24  0.0.0.0                  0    100  64512 ?
```

**Comment by devasmith** [comment]
Hi. Sorry for the late reply. It would be sufficient if we could get a total count of advertised-prefixes like for example as with frr_bgp_prefixes_active.

**Comment by tynany** [comment]
Apologies for the delay. I have implemented the below metric in v0.2.4. You can find the binaries under releases -- https://github.com/tynany/frr_exporter/releases/tag/v0.2.4

```
frr_bgp_peer_prefixes_advertised_count_total
```

Example output:
```
# HELP frr_bgp_peer_prefixes_advertised_count_total Number prefixes advertised.
# TYPE frr_bgp_peer_prefixes_advertised_count_total gauge
frr_bgp_peer_prefixes_advertised_count_total{afi="ipv4",local_as="64800",peer="swp1",peer_as="64801",safi="unicast",vrf="default"} 3
```
